samples/msal-node-samples/client-credentials-distributed-cache/src/AxiosHelper.ts

```diff
@@ -12,16 +12,18 @@ class AxiosHelper {
      * @param endpoint
      * @param accessToken
      */
-    static async callEndpointWithToken(endpoint: string, accessToken: string | undefined): Promise<any> {
-        const options = {
-            headers: {
-                Authorization: `Bearer ${accessToken}`
-            }
-        };
-
+    static async callEndpointWithToken(endpoint: string, accessToken?: string, params?: Record<string, string>): Promise<any> {
```

Comment:
Consider renaming to `callDownstreamApi` since `Endpoint` is quite overloaded in OAuth space.,

Reply:
done!